docs/src/basics/nomenclature.md

```diff
@@ -0,0 +1,53 @@
+
+_Workflow Variable_:  A workflow variable represents the execution state of a workflow instance.  The variables capture business process parameters which are input and output of various stages of the workflow instance and which also influence process flow execution.
+
+_Workflow Deployment_: a workflow definition cannot execute unless the broker is aware of its existence. This is done by deploying a committing a workflow definition to the broker. The act of pushing of broker is referred to as workflow deployment.
+
+_Broker_:  A broker is an instance of a Zeebe installation which executes workflows and manages workflow state.  A single broker will be installed on a single machine.
+
+_Cluster_:  A cluster represents a configuration of one or more brokers collaborating to execute workflows.  Each broker in a cluster acts as a leader or a follower.  
+
+_Leader_:  In a clustered environment, one broker tis responsible for workflow execution and housekeeping.  Housekeeping includes, taking snapshots, replication and running exports for a given partition.
+
+_Follower_:  In a clustered environment, a broker which is not a leader is a follower of a given partition.  A follower assumes leader responsibilities when the leader goes down.
+
+_Record_: A Zeebe record is a collection of workflow variables and other state transitions of an executing workflow instance at a given point in time.  During execution of a workflow instance numerous records will be generated to capture the various state transitions from start to workflow instance completion.
+
+_Event_:  Within the context of Zeebe, an event represent a state transition associated with an executing workflow instance.  Events include variable changes, changes to transition elements in the workflow model, etc.   An event will be represented by a timestamp, the variable name and variable value.  These events are stored in an append-only log.
+
+_Log_:  An ordered sequence of events written in persistent storage. This log file is only appended-to and is stored on disk.  
+
+_Segment_:  Data within a log file is segments.  Segments are defined by time period.???
+
+_Partition_:  A partition represents a logical grouping of data in as Zeebe broker.  This data includes workflow instance variables and events generated by Zeebe.  Partitions are determined by ???? 
+
+_Replication_:  In a clustered Zeebe installation configured with more than one broker, data in a partition is copied to the other brokers.  This act of copying data in partitions from one broker to another is also referred to as replication.
```

Comment:
Note that the data is replicated from the leader to the followers of a partition.

Reply:
Noted.